app/controllers/v2/link_sets_controller.rb

```diff
@@ -23,6 +27,12 @@ def get_linked
 
   private
 
+    def with_drafts?
+      # Cast the `with_drafts` query param to a real boolean, and default to
+      # `true` to preserve existing behaviour
+      ActiveModel::Type::Boolean.new.cast(params.fetch(:with_drafts, true))
```

Comment:
Nah no convention, I did some googling after commenting before and looks like we need something like this here, bit of a bummer really since ActiveModel seems a totally unrelated dependency in this context but I don't know a better way.